### DISCUSSION TITLE
Error on installation

### DISCUSSION THREAD

**Discussion by quangkieu** [valinet/ExplorerPatcher#546] [Q&A] [answered]
As the title, I try to use the beta and actual release but no luck
So, I download the source and compile by document, and this is the error I got
![image](https://user-images.githubusercontent.com/4308312/145669483-da635294-3092-4da4-b60d-e9c8606e0bb0.png)
So, the symptom is the same, it just now I have the error screen:
No installer wizard or anything
Explorer crashed/restarted (expected)
Win 10 taskbar back, wifi, sound and everything back notification back.
But I could not find any GUI of ExplorerPatcher anywhere.
![image](https://user-images.githubusercontent.com/4308312/145669794-de684834-2bb9-4aa1-8478-d1a6c0bec2f0.png)

This is what in installer location
![image](https://user-images.githubusercontent.com/4308312/145669659-da160162-1924-4aec-be35-39cc0f670708.png)

**Comment by valinet** [comment, marked as answer]
So, what’s not working after all? Right click taskbar - Properties should open the configuration GUI.

**Reply by quangkieu** [reply]
Oh, the setting in there. Sorry, I could not find any picture referencing where the setting is. I thought would be some notification icon. As when I try to run the ep_setup in installation folder, it just remove the app and not pop-up some gui

**Reply by valinet** [reply]
Btw, I have now just fixed that stack corruption/stack buffer overflow problem and have just pushed the commit.
